.github/workflows/docker_cpu_nightly.yaml

```diff
@@ -0,0 +1,28 @@
+# GH Actions
+name: Docker-CPU-Nightly
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: '0 6 * * *' # 6 AM UTC
+
+jobs:
+  Build-Cpu:
```

Comment:
All of the other jobs are named `Build`, is this for any particular reason. For local testing with `act`, it helps to give these different names. This either needs to be changed to make it consistent with the other build files, or the other build files should be changed to disambiguate them.